src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerArgumentCollection.java

```diff
@@ -107,11 +108,12 @@ public ReadThreadingAssembler createReadThreadingAssembler() {
     public SmithWatermanAligner.Implementation smithWatermanImplementation = SmithWatermanAligner.Implementation.JAVA;
 
     /**
-     * (BETA feature) The reference confidence mode makes it possible to emit a per-bp or summarized confidence estimate for a site being strictly homozygous-reference.
-     * This is similar to the HaplotypeCaller reference confidence/GVCF mode. See https://software.broadinstitute.org/gatk/documentation/article.php?id=4017 for information about GVCFs.
+     * The reference confidence mode makes it possible to emit a per-bp or summarized confidence estimate for a site being strictly homozygous-reference.
+     * See https://software.broadinstitute.org/gatk/documentation/article.php?id=4017 for information about GVCFs.
+     * This is a BETA FEATURE for Mutect2 similar to the HaplotypeCaller reference confidence/GVCF mode.
```

Comment:
Recommend rewording slightly to put "Mutect2" at the start of the sentence before the word "BETA": "For Mutect2, this is a BETA feature that functions similarly to the HaplotypeCaller reference confidence/GVCF mode."